docs/source/conf.py

```diff
@@ -15,6 +15,15 @@
 import apptools
 import enthought_sphinx_theme
 
+# Apptools offers an envisage plugin that requires importing from
+# envisage.
+try:
+    import envisage  # noqa
+except ImportError:
+    autodoc_mock_imports = [
+        'envisage',
+    ]
+
```

Comment:
Nit: Although this includes an import and import tends to go to the top, this is still a configuration specific to the autodoc extension.
Normally extension options are placed after "General configuration".
Following current convention, this would go under a section "# Options for autodoc"?